docs/user-guide/content-map-toolbar.md

```diff
@@ -0,0 +1,16 @@
+# Content Map Toolbar
+**********************
+
+As soon as a map content is added, the Content Map Toolbar appears on top of the map component:
```

Comment:
As soon as a map content is added, the Map Content Toolbar appears on top of the map component: